R/write.R

```diff
@@ -72,7 +74,10 @@ write_union <- function(path, lines, quiet = FALSE) {
 #' @param overwrite Force overwrite of existing file?
 #' @export
 write_over <- function(path, lines, quiet = FALSE, overwrite = FALSE) {
-  stopifnot(is.character(lines), length(lines) > 0)
+  check_name(path)
+  check_character(lines)
```

Comment:
I think we lost the check that lines has at least one element FWIW.

Reply:
Hmmm good point. Should `check_character()` perhaps have a `length` or `min_length` argument? Or should there be a separate `check_length()` function?

Reply:
I think I've looked for that myself, when converting other packages, and I did not find anything.

Reply:
I'll open an issue in `rlang`, I think it would be useful

Reply:
https://github.com/r-lib/rlang/issues/1618

Reply:
Added back a `stopifnot()` for now